src/networking/httpService.ts

```diff
@@ -13,6 +13,75 @@ async function get<T = any>(endpoint: string): Promise<T> {
   }
 }
 
+async function post<T = any>(
+  endpoint: string,
+  title: string,
+  description: string,
+): Promise<T> {
+  try {
+    const response = await fetch(baseUrl + endpoint, {
+      body: JSON.stringify({
+        description: description,
+        title: title,
```

Comment:
Probably you would like to do it more generic than this.

```suggestion
        body,
```